INN/umbrella-caribbean
# 20 two column list

## Changes

This pull request makes the following changes:

- styles tables to match the two-column mockups
- styles headings within the entry content to use the display font and `@black` color

<img width="821" alt="Screen Shot 2019-11-26 at 21 16 13" src="https://user-images.githubusercontent.com/1754187/69687831-43fac900-1092-11ea-85d4-c013b949c04e.png">
<img width="336" alt="Screen Shot 2019-11-26 at 21 18 27" src="https://user-images.githubusercontent.com/1754187/69687849-5117b800-1092-11ea-82e6-1a0afd733350.png">


## Why



For #20

## Testing/Questions

Features that this PR affects:

- table block
- heading block


Questions that need to be answered before merging:

- [x] do we need to test this with more columns?

Steps to test this PR:

1. Put one of each level of "heading" block in a post
2. Put a table in a post, and fill it with content. Check the box to include a table header. If you want all-caps text in the header, you'll need to type it that way.
